src/components/home/elements/SendToken.vue

```diff
@@ -616,8 +623,14 @@ export default {
             }
             return common.getLength(this.description) <= TRANSFER_ATTACHMENT_BYTE_LIMIT
         },
+        getAddressFromDataObject() {
+            return this.dataObject.stored_tx.senderPublicKey ? this.account.convertPublicKeyToAddress(this.dataObject.stored_tx.senderPublicKey, this.dataObject.network_byte) : this.coldAddress
```

Comment:
confuse about the default `this.coldAddress` if `this.dataObject.stored_tx.senderPublicKey` can not be accessed.

Reply:
![image](https://user-images.githubusercontent.com/32302909/76283810-9759f580-62d6-11ea-94ee-f5caceaa0cfa.png)
In initData, the default coldAddress has been handled.
`getAddressFromDataObject` is used in html page and will be called when SendToken module is initialized(not shown yet).`senderPublicKey` has not yet been entered by the user, so this.account.convertPublicKeyToAddress`` will throw error if it gets empty parameter.

Reply:
Let me explain my confusion in more detail here.
1. compare with the `showAddress()` functions in other files(e.g. why only here senderPublicKey can not be accessed?). You explain that it is related to initialization, but it is not the reason that the two files' code are not consistent.

2. if `senderPublicKey` can not be accessed, then refer the issue #385 , it is not the correct solution to the issue. In the issue, if I understand clearly, that we ask to show the exact value which passed to the cold wallet. But here, if `senderPublicKey` can not be accessed, it should return an error in a cold wallet and will not show the page listed here(cold page = 4?), right? 
The correct logic here is that the value showed in the confirm page should all from `DataObject` directly, no extra judgment.

Reply:
of course, we can use `getAddressFromDataObject` in other places, but my understanding of the usage of the cold pages are
1. `coldPageId===2`, confirm the data before passing to the cold wallet, if `senderPublicKey` incorrect, it should fail.
2. `coldPageId===4`, double confirm before a push to the node. already received a signature from the cold wallet, if `senderPublicKey` incorrect, it should fail in cold wallet.
3. `coldPageId===5`, success page, all information from `DataObject` should be correct, right?

refer the usage~
```javascript
          <TokenConfirm :address="getAddressFromDataObject"
                        :recipient="dataObject.stored_tx.functionData.recipient"
                        :amount=inputAmount(dataObject.stored_tx.functionData.amount)
                        :fee="fee"
                        :description="dataObject.stored_tx.attachment"
                        :tx-type="'Send Token'">
          </TokenConfirm>
```
and
```javascript
          <TokenSuccess class="tokenSucced"
                        :address="getAddressFromDataObject"
                        :recipient="dataObject.stored_tx.functionData.recipient"
                        :amount=inputAmount(dataObject.stored_tx.functionData.amount)
                        :description="dataObject.stored_tx.attachment"
                        :fee="fee"
                        :tx-type="'Send Token'">
          </TokenSuccess>
```